src/xrpld/rpc/handlers/ServerInfo.cpp

```diff
@@ -62,6 +63,248 @@ class ServerDefinitions
     {
         return defs_;
     }
+    static Json::Value
+    parseLedgerEntryFlags()
+    {
+        Json::Value solution = Json::objectValue;
+        std::string const ledgerFormatFile{
```

Comment:
Source code regex won't work on production systems. It actually doesn't even work on my machine, because my build directory isn't in my source directory - 
```sh
% ./rippled server_definitions
Loading: "../rippled.cfg"
2025-Jul-25 21:07:08.911682 UTC HTTPClient:NFO Connecting to 127.0.0.1:5005

{
   "result" : {
      "error" : "internal",
      "error_code" : 73,
      "error_message" : "Internal error.",
      "request" : {
         "api_version" : 1,
         "command" : "server_definitions"
      },
      "status" : "error"
   }
```
Also, none of the tests pass.